pkgs/applications/graphics/gimp/default.nix

```
@@ -115,6 +116,7 @@ in stdenv.mkDerivation rec {
     libwebp
     libheif
     python
+    python2Packages.pygtk
```

```suggestion
    python2.pkgs.pygtk
```

---

I think this will probably be necessary, since `nix-build -E '(import ./. {}).python2.withPackages (pp: [ (pp.pygtk.overrideAttrs (attrs: { outputs = ["out" "dev"]; })) ])'` does not include the pkg-config file.

But it should use the following to reduce the chance of inconsistency between `python2.pkgs` `python2Packages` when someone overrides one of them:

```suggestion
    python2.pkgs.pygtk # For pkg-config file, since the environment in `python` does not expose it.
```

---

This seems to work:

```shell
nix-build -E 'let pkgs = import ./. {}; in (pkgs.python2.buildEnv.override { extraOutputsToInstall = [ "dev" ]; extraLibs = [ (pkgs.python2.pkgs.pygtk.overrideAttrs (attrs: { outputs = ["out" "dev"]; })) ]; postBuild = "moveToOutput lib/pkgconfig $dev"; }).overrideAttrs (attrs: { outputs = ["out" "dev"]; })'
```

so maybe replace `python2.withPackages` by

```nix
# Make pkg-config files available in a separate output:
(python2.buildEnv.override {
  extraOutputsToInstall = [ "dev" ];
  extraLibs = [
    python2.pkgs.pygtk
  ];
  postBuild = ''
    moveToOutput lib/pkgconfig "$dev"
  '';
}).overrideAttrs (attrs: {
  outputs = ["out" "dev"];
})
```

---

I guess we might want to fix it in `buildEnv` so that we do not need to manually replicate `multiple-outputs.sh` hook in `postBuild`. See https://github.com/NixOS/nixpkgs/pull/123535.

---

php's buildEnv seem to differ a way from python, I don't really understand what should I change